velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -66,6 +70,26 @@ TEST_F(DateTimeFunctionsTest, yearDate) {
   EXPECT_EQ(1920, year(DATE()->toDays("1920-01-01")));
 }
 
+TEST_F(DateTimeFunctionsTest, weekDate) {
```

Comment:
When the input is invalid date, is the behavior the same as Spark's?

Reply:
spark just test null for invalid input, which can't be tested in velox, no other exception test.

Reply:
What about `1997-02-29` which is invalid?

Reply:
It's coverd by velox function ` daysSinceEpochFromDate`
```cpp
C++ exception with description "Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: Date out of range: 1997-2-29
Retriable: False
Function: daysSinceEpochFromDate
File: ../../velox/type/TimestampConversion.cpp
Line: 455
Stack trace:
```